specification/mysql/resource-manager/Microsoft.DBforMySQL/FlexibleServers/readme.md

```diff
@@ -95,6 +95,407 @@ suppressions:
+- preview/2021-12-01-preview/Configurations.json
+- preview/2021-12-01-preview/Databases.json
+- preview/2021-12-01-preview/FirewallRules.json
+- preview/2021-12-01-preview/FlexibleServers.json
+- preview/2021-12-01-preview/LogFiles.json
+- preview/2021-12-01-preview/ServiceOperations.json
+- preview/2021-12-01-preview/AzureADAdministrator.json
+```
+
+### Tag: package-flexibleserver-2022-09-30-preview
+
+These settings apply only when `--tag=package-flexibleserver-2022-09-30-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-flexibleserver-2022-09-30-preview'
+input-file:
+- preview/2021-12-01-preview/AzureADAdministrator.json
+- preview/2022-09-30-preview/Backups.json
+- preview/2022-09-30-preview/BackupAndExport.json
+- preview/2021-12-01-preview/Configurations.json
+- preview/2021-12-01-preview/Databases.json
+- preview/2021-12-01-preview/FirewallRules.json
+- preview/2022-09-30-preview/FlexibleServers.json
+- preview/2021-12-01-preview/LogFiles.json
+- preview/2022-09-30-preview/ServiceOperations.json
+- common-types/v1/common-types.json
+suppressions:
+  - code: PropertiesTypeObjectNoDefinition
+    from: common-types.json
+    reason: This will be fixed in new versions.
+```
+
+### Tag: package-flexibleserver-2023-06-01-preview
+
+These settings apply only when `--tag=package-flexibleserver-2023-06-01-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-flexibleserver-2023-06-01-preview'
+input-file:
+- preview/2023-06-01-preview/AzureADAdministrator.json
+- preview/2023-06-01-preview/Backups.json
+- preview/2023-06-01-preview/BackupAndExport.json
+- preview/2023-06-01-preview/Configurations.json
+- preview/2023-06-01-preview/Databases.json
+- preview/2023-06-01-preview/FirewallRules.json
+- preview/2023-06-01-preview/FlexibleServers.json
+- preview/2023-06-01-preview/LogFiles.json
+- preview/2023-06-01-preview/ServiceOperations.json
+- common-types/v1/common-types.json //todo
+suppressions:
+  - code: PostOperationAsyncResponseValidation
+    from: FlexibleServers.json
+    reason: This check is optional.
+  - code: PropertiesTypeObjectNoDefinition
+    from: common-types.json
+    reason: This will be fixed in new versions.
+```
+
+### Tag: package-flexibleserver-2023-06-01-preview-new
+
+These settings apply only when `--tag=package-flexibleserver-2023-06-01-preview-new` is specified on the command line.
+
+``` yaml $(tag) == 'package-flexibleserver-2023-06-01-preview-new'
+input-file:
+- preview/2023-06-01-preview/AzureADAdministrator.json
+- preview/2023-06-01-preview/Backups.json
+- preview/2023-06-01-preview/BackupAndExport.json
+- preview/2023-06-01-preview/Configurations.json
+- preview/2023-06-01-preview/Databases.json
+- preview/2023-06-01-preview/FirewallRules.json
+- preview/2023-06-01-preview/FlexibleServers.json
+- preview/2023-06-01-preview/LogFiles.json
+- preview/2021-12-01-preview/ServiceOperations.json
+- common-types/v1/common-types.json //todo
```

Comment:
Do we have common APIs (not a model definition) within common-types folder? I don't think we need to explicitly add common-types into the tag, as those definition should be internally referred by the APIs?